velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +91,24 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseWeibullCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double a, double b, double p) {
+    constexpr double kInf = std::numeric_limits<double>::infinity();
```

Comment:
This can be static variable ... static constexpr double kInf.

Reply:
Fixed.